### ISSUE TITLE
Button icons aren't rendering in Firefox 72.0.1

### ISSUE THREAD

**Issue by dessalines** [jellyfin/jellyfin#2284]
On my local jellyfin installation, none of the button icons are rendering since I updated Firefox to `72.0.1`

![](https://i.imgur.com/2paWqcP.png)

```
Version: 10.4.3
Operating System: Linux
Architecture: X64
```

**Comment by nyanmisaka** [comment]
Make sure the checkbox in `Firefox -> Options -> General -> Fonts&Colors -> Advanced -> Allow pages to choose their own fonts, instead of your selections above` is selected.

**Comment by dessalines** [comment]
This is true, I'm using a custom font. But shouldn't this be using an externalized icon library like [fork-awesome](https://forkaweso.me/) to make sure that the icons work on all browsers and platforms?
